.travis.yml

```diff
@@ -52,11 +61,35 @@ matrix:
      sudo: true
    - env: TOXENV=py35-dj110head-postgres-noelasticsearch
      python: 3.5
+   - env: TOXENV=py35-dj111-sqlite-noelasticsearch
+     python: 3.5
+   - env: TOXENV=py35-dj111-postgres-noelasticsearch
+     python: 3.5
+   - env: TOXENV=py35-dj111-mysql-noelasticsearch
+     python: 3.5
+   - env: TOXENV=py35-dj111-postgres-elasticsearch
+     python: 3.5
+   - env: TOXENV=py35-dj111-postgres-elasticsearch2 INSTALL_ELASTICSEARCH2=yes
+     python: 3.5
+     sudo: true
+   - env: TOXENV=py35-dj111-postgres-elasticsearch5 INSTALL_ELASTICSEARCH5=yes
+     python: 3.5
+     sudo: true
   allow_failures:
+    - env: TOXENV=py27-dj110-mysql-elasticsearch5 INSTALL_ELASTICSEARCH5=yes
```

Comment:
This change about Django 1.10 is confusing as part of this PR, especially because the build is passing, yet it's in allowed_failures -- perhaps a comment could clarify that if it's correct.